### PR TITLE
Issue#2591

### PR DESCRIPTION
**Pull request by ArturGoldyn**
Added functionality + created tests for it.
Also, php-cs-fixer.phar has been run against all code.
It was suprisingly easy task, because if you skip method name in process isolation routine it runs the whole class in isolation.

### CODE REVIEW

File: src/Util/Test.php [hunk 1]
```diff
@@ -967,7 +981,7 @@ private static function getBooleanAnnotationSetting($className, $methodName, $se
             }
         }
 
-        return null;
+        return;
```
Comment (GrahamCampbell) [inline src/Util/Test.php]: this statement is dead code

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Util/Test.php (hunk 1 above)
Comment (ArturGoldyn) [inline src/Util/Test.php]: And is this the cause of failing Appveyor?

**Comment by ArturGoldyn** [comment]
Closing this for now, I'll try it without this php-cs-fixer run not on my files.

File: src/Util/Test.php (hunk 1 above)
Comment (GrahamCampbell) [inline src/Util/Test.php]: Doubt it. Not looked though.

Comment (ArturGoldyn) [inline src/Util/Test.php]: I added a new pull request, without this change.
https://github.com/sebastianbergmann/phpunit/pull/2664

**Pull request closed without merging**